sdks/python/apache_beam/runners/worker/statesampler_fast.pyx

```diff
@@ -109,25 +80,18 @@ cdef class StateSampler(object):
   cdef pythread.PyThread_type_lock lock
 
   cdef public int64_t state_transition_count
-  cdef int64_t time_since_transition
+  cdef int64_t _time_since_transition
 
   cdef int32_t current_state_index
 
-  def __init__(self, prefix, counter_factory,
-      sampling_period_ms=DEFAULT_SAMPLING_PERIOD_MS):
-
-    # TODO(pabloem): Remove this once all dashed prefixes are removed from
-    # the worker.
-    # We stop using prefixes with included dash.
-    self.prefix = prefix[:-1] if prefix[-1] == '-' else prefix
-    self.counter_factory = counter_factory
-    self.sampling_period_ms = sampling_period_ms
+  def __init__(self, *args):
+    #TODO(pabloem): Figure out how to pass arguments without errors.
```

Comment:
<!--thread_id:cc_161898593_t; commit:4c27aa0fbbbc31e24e3b8d6bc04128b44d18b9ec; resolved:1-->
<!--section:context-quote-->
> **robertwb** wrote:
> What errors were you getting?

<!--section:body-->
hmmm this is fixed now.